examples/dgmulti_3d/elixir_euler_tensorWedge.jl

```diff
@@ -0,0 +1,59 @@
+using OrdinaryDiffEq
+using Trixi, StartUpDG
+using LinearAlgebra
+
+###############################################################################
+equations = LinearScalarAdvectionEquation3D(1.0, 1.0, 1.0)
+
+initial_condition = initial_condition_convergence_test
+
+# Define the polynomial degrees for the polynoms of the triangular base and the line
+# of the tensor-prism
+tensor_polydeg=(2,3)
+
+dg = DGMulti(element_type = Wedge(),
+            approximation_type = Polynomial(),
+            surface_flux = flux_lax_friedrichs,
+            polydeg=(2,3))
```

Comment:
```suggestion
            polydeg=(2, 3))
```
Did you want to use the `tensor_polydeg` defined above here?